components/schemes/Color/src/MDCSemanticColorScheme.m

```diff
@@ -69,6 +84,18 @@ - (instancetype)initWithDefaults:(MDCColorSchemeDefaults)defaults {
         _onSurfaceColor = ColorFromRGB(0xFFFFFF);
         _onBackgroundColor = ColorFromRGB(0xFFFFFF);
         break;
+      case MDCColorSchemeDefaultsMaterial201907:
+        _primaryColor = DynamicColor(ColorFromRGB(0x6200EE), ColorFromRGB(0xBB86FC));
+        _primaryColorVariant = DynamicColor(ColorFromRGB(0x3700B3), ColorFromRGB(0x3700B3));
```

Comment:
Should this be the same in both cases?

Reply:
Same in this one. Reference: https://material.io/design/color/dark-theme.html#ui-application